addons/mail/static/src/components/emoji_list/emoji_list.xml

```diff
@@ -2,12 +2,21 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.EmojiList" owl="1">
-        <div class="o_EmojiList px-1" t-attf-class="{{ className }}" t-ref="root">
-            <t t-if="emojiListView">
-                <t t-foreach="emojiListView.emojiViews" t-as="emojiView" t-key="emojiView.localId">
-                    <Emoji emojiView="emojiView"/>
+        <div>
+            <div>
+                <t t-if="props.record">
+                    <t t-foreach="props.record.emojiCategoryViews" t-as="emojiCategoryView" t-key="emojiCategoryView.localId">
+                        <EmojiCategory emojiCategoryView="emojiCategoryView"/>
+                    </t>
                 </t>
-            </t>
+            </div>
+            <div  class="o_EmojiList px-1" t-attf-class="{{ className }}" t-ref="root">
+                <t t-if="props.record">
+                    <t t-foreach="props.record.emojiViews" t-as="emojiView" t-key="emojiView.localId">
+                        <Emoji emojiView="emojiView"/>
+                    </t>
+                </t>
+            </div>
```

Comment:
Root node should be `o_EmojiList`.
You should avoid introducing arbitrary `div`.

I don't think `EmojiCategoryViews` and `EmojiViews` should be immediately next to each other.
It looks like some concepts are missing, like "EmojiCategoryBar" or `EmojiListView` (here as `EmojiListView` being the content showing the emojis, actually `EmojiListView` should be renamed to `EmojiPickerView`).